.github/CODEOWNERS

```
@@ -0,0 +1 @@
+* @molawson @sheck
```

In light of the recent creation of a "Balto Maintainers" team:

```suggestion
* @planningcenter/balto-maintainers
```